comid-triples.cddl

```diff
@@ -0,0 +1,142 @@
+concise-mid-tag = {
+  ; TODO(tho) language
+  ; TODO(tho) tag-id
+  ; TODO(tho) module-name
+  ; TODO(tho) entity
+  ; TODO(tho) linked-tags
+  comid.claims => claims-map
+  ; TODO(tho) extension
+}
+
+claims-map = non-empty<{
+  comid.reference-claims => one-or-more<reference-claim-record>
+  comid.endorsed-claims => one-or-more<endorsed-claim-record>
+  comid.identity-claims => one-or-more<identity-claim-record>
+  * $$claims-map-extension
+}>
+
+; REFERENCE-VALUE triple
+; "target environment $TE" "has reference measurements" "$RV"
+reference-claim-record = {
+  target-environment: environment-map 
+  reference-values: values-map
```

Comment:
Use array in vales-map instead of singleton!

Reply:
```suggestion
  reference-values: one-or-more<values-map>
```